From 746b4be569bbd6d2ced45640eefff8b445f6b4ab Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Mon, 12 Oct 2015 15:18:49 +0800
Subject: [PATCH 0554/5242] MLK-11691 ARM: dts: imx6sl-evk: add pxp v4l2
 output support

commit  8041165c5a3a7f17cdf53d3e1a8b862abe1a1976 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add pxp v4l2 output support for imx6sl evk board.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sl-evk.dts |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sl-evk.dts b/arch/arm/boot/dts/imx6sl-evk.dts
index 7de8f68..6825173 100644
--- a/arch/arm/boot/dts/imx6sl-evk.dts
+++ b/arch/arm/boot/dts/imx6sl-evk.dts
@@ -35,6 +35,11 @@
 		};
 	};
 
+	pxp_v4l2_out {
+		compatible = "fsl,imx6sl-pxp-v4l2";
+		status = "okay";
+	};
+
 	reg_usb_otg1_vbus: regulator-usb-otg1-vbus {
 		compatible = "regulator-fixed";
 		regulator-name = "usb_otg1_vbus";
-- 
1.7.9.5

